### PR TITLE
feat: Add pathname to prev page events

### PR DESCRIPTION
**Pull request by robbie-c**
## Changes

Follow up from https://github.com/PostHog/posthog-js/pull/773, looks like I did need this

## Checklist
- [x] Tests for new code (see [advice on the tests we use](https://github.com/PostHog/posthog-js#tiers-of-testing))
- [x] Accounted for the impact of any changes across different browsers

### CODE REVIEW

File: src/page-view.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import * as utils from './utils'
```
Comment (benjackwhite) [inline src/page-view.ts:1]: Small thing from the last PR - do we need to import * ? Why not scope the vars (helps in the future to understand unused parts)

File: src/page-view.ts [hunk 2]
```diff
@@ -24,15 +25,21 @@ interface ScrollProperties {
...
+}
+
 export class PageViewManager {
     _pageViewData: PageViewData | undefined
     _hasSeenPageView = false
 
     _createPageViewData(): PageViewData {
-        return {}
+        return {
+            pathname: utils.window.location.pathname,
```
Comment (benjackwhite) [inline src/page-view.ts:38]: I assume we need this to query on? Do we perhaps want to just have a full url which allows any kind of grouping filtering later on?

**Review by benjackwhite — APPROVED** [review]
Approved with a couple of optional comments

File: src/page-view.ts (hunk 1 above)
Comment (robbie-c) [reply src/page-view.ts:1]: Hmm it might be needed based on how I set up the mocking in the tests, I'll play around with it some more

Comment (robbie-c) [reply src/page-view.ts]: Fixed

**Comment by robbie-c** [comment]
Merging as tests passed on https://github.com/PostHog/posthog-js/pull/780

File: src/page-view.ts [hunk 3]
```diff
@@ -24,15 +25,21 @@ interface ScrollProperties {
     $prev_pageview_max_content_percentage?: number
 }
 
+interface PageViewEventProperties extends ScrollProperties {
+    $prev_pageview_pathname?: string
```
Comment (Twixes) [inline src/page-view.ts:29]: We might want to update `isURLNormalizeable()` in the main repo to include this property

**Review by Twixes — APPROVED** [review]

**Pull request merged**